components/console-backend-service/internal/domain/k8s/resourcequota_service_test.go

```diff
@@ -36,6 +36,42 @@ func TestResourceQuotaResolver_ListSuccess(t *testing.T) {
 	assert.Len(t, result, 2)
 }
 
+func TestResourceQuota_Create(t *testing.T) {
+	const (
+		namespace = "production"
+	)
+	var testCases = []struct {
```

Comment:
You can simplify that code:
``` go
for caseName, test := range []struct{
                caseName               string
		namespace              string
		name                   string
		resourceLimitsMemory   string
		resourceRequestsMemory string
		success                bool
	} {
        ...test cases
        } {
             t.Run(caseName, func(t *testing.T) {
                ...logic for test
             }
        }
```

Please also don't use var, but write `:=` as assignment 😄

Reply:
Not sure if I'd call it simplification ;)

Reply:
For my, that code is simplifier :D